cypress/integration/ordinal-measures.spec.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  loadChartInLocalStorage,
+  waitForChartToBeLoaded,
+} from "../charts-utils";
+import testOrd507 from "../fixtures/test-ord-507-chart-config.json";
+
+import selectors from "./selectors";
+
+Cypress.on("uncaught:exception", (err) => {
+  if (err.message.includes("> ResizeObserver loop")) {
+    return false;
+  }
+});
+
+beforeEach(() => {
+  cy.viewport("macbook-15");
+});
+
+describe("viewing a dataset with only ordinal measures", () => {
+  const key = "ePUgYyo622qS";
+  const config = testOrd507;
+
+  it("should retrieve dimension values properly", () => {
+    cy.visit(
```

Comment:
No need to loadChartInLocalStorage here ?

Reply:
Nope, here we only look at dataset preview :)